src/tooling/piral-cli/src/common/scripts.ts

```diff
@@ -29,11 +28,13 @@ export function runScript(script: string, cwd = process.cwd(), output: NodeJS.Wr
       resolveWinPath('AppData', 'npm'),
       resolveWinPath('ProgramFiles', 'nodejs'),
       resolveWinPath('ProgramFiles(x86)', 'nodejs'),
-      ...env.PATH.split(';'),
+      ...env.Path.split(';'),
```

Comment:
Could we do `...(env.Path || env.PATH || '').split(';')` here? When I noticed on my own Windows machine that `env.Path` may in fact be `undefined` I got a bit nervous here.

Rest looks awesome - much appreciated!

Reply:
Added defensive coding around usage of the Path/PATH property.